build.sh

```diff
@@ -97,20 +151,25 @@ echo "#define STRING_DISTRIBUTION_DATE" `date '+"%Y-%m-%d %H:%M"'` >>"$OUTFILE"
 )
 
 # If /build/ exists, remove.
-if [ -d "$HERE/build/" ]; then
-  echo "Build directory exists, removing..."
-  rm -rf $HERE/build/
+if [ ! "$OPERATING_SYSTEM" = "Linux" ]; then
+  if [ -d "$HERE/build/" ]; then
+    echo "Build directory exists, removing..."
+    rm -rf $HERE/build/
+  fi
 fi
 
 echo $VERSION
 echo ""
 
-case "$(uname -s)"
+case "$OPERATING_SYSTEM"
   in Darwin)
     ARDUINO_EXEC="/Applications/Arduino.app/Contents/MacOS/Arduino $COMMAND \"$HERE/Marlin/Marlin.ino\" --pref build.path=\"$HERE/build/\" --pref board=rambo $PORT_ARG"
-    ARDUINO_DEP="/Applications/Arduino.app/Contents/Java/hardware/arduino/avr/"
+    ARDUINO_DEP="/Applications/Arduino.app/Contents/Java/hardware/arduino/avr"
   ;; Linux)
-    # Future support for Linux
+    ARDUINO_DEP="/usr/share/arduino/hardware/arduino"
+    ARDUINO_EXEC_COMPILE="ino build -m mega2560"
+    ARDUINO_EXEC_UPLOAD="ino upload -m mega2560"
+    #ARDUINO_EXEC_UPLOAD="/usr/share/arduino/hardware/tools/avrdude -q -q -C /usr/share/arduino/hardware/tools/avrdude.conf -U flash:w:firmware.hex:i -v -p atmega2560 -b 115200 -c stk500v2 -D"
```

Comment:
I was going to but thought that this might be useful for the future how `avrdude` works. Is there some place I can store this for later that's not inside the script itself?